### PR TITLE
Use Lux footer

### LINKED ISSUE #4087: Use LUX to provide Site Footer
### Current display
We are displaying the library legacy footer.

### Desired display
Move to the new footer that is part of the current release of lux. 

### Acceptance criteria

- [x] Lux is included in Orangelight
- [x] The new footer is displayed. 
- [ ] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
* [Instructions for adding Lux to a rails application](https://github.com/pulibrary/lux-design-system/blob/main/docs/adding_lux.md#adding-lux-to-a-viterails-project)

### PR DESCRIPTION
**Pull request by maxkadel**
Closes #4087

### CODE REVIEW

**Review by christinach — APPROVED** [review]
Thanks @maxkadel

**Review by christinach — COMMENTED** [review]
@maxkadel can you take a look in the X social icon? Maybe there is a css class in the app that changes it?

**Comment by maxkadel** [comment]
> @maxkadel can you take a look in the X social icon? Maybe there is a css class in the app that changes it?

Thanks for catching that! Yes, there was bootstrap 4 css on the svg element that was making it wonky.

File: app/assets/stylesheets/shame.scss [hunk 1]
```diff
@@ -109,4 +109,9 @@
 // Bootstrap 5 will include the .pe-none class
 .pe-none {
   pointer-events: none;
-}
\ No newline at end of file
+}
+
+// bootstrap 4 sets vertical-align to middle, which makes the "X" icon in the footer look wrong
+svg {
+  vertical-align: unset;
```
Comment (christinach) [inline app/assets/stylesheets/shame.scss:116]: 👍🏻

File: spec/system/footer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe 'the shared footer', type: :system, js: true do
+  it 'renders the footer' do
+    visit '/'
+    expect(page).to have_link('Accessibility Help')
+    expect(page).to have_selector('.lux-library-logo')
+    expect(page).to have_link('For Library Staff')
```
Comment (christinach) [inline spec/system/footer_spec.rb:9]: 👍🏻

**Review by christinach — APPROVED** [review]
Thanks @maxkadel and @sandbergja

**Pull request merged**